echo-notifications/src/test/groovy/com/netflix/spinnaker/echo/config/GithubConfigSpec.groovy

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.echo.config
+
+import spock.lang.Specification
+import spock.lang.Subject
+import retrofit.Endpoint
+
+class GithubConfigSpec extends Specification {
```

Comment:
These tests don't actually do anything useful, and don't actually test that Spring is injecting the right value. Please remove.

Reply:
Thank you!
The intention of the test is to ensure that the "endpoint" attribute is not removed, because this happened and broke our process. I think the test helps this not happen again.
Of course, if you have any suggestions, it is welcome!

Reply:
Got it - I see in the history where it was removed.

In that case, I would still advocate for removing the second test, as it appears to be a [change detector test](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html).

Reply:
Done! Thanks.